src/main/java/com/cleanroommc/groovyscript/compat/mods/jei/JustEnoughItems.java

```diff
@@ -35,10 +42,43 @@ public void removeAndHide(IIngredient ingredient) {
         JeiPlugin.hideItem(ingredient.getMatchingStacks());
     }
 
+    public void removeAndHide(IIngredient... ingredients) {
+        for (IIngredient ingredient : ingredients) {
+            if (IngredientHelper.isEmpty(ingredient)) {
+                GroovyLog.msg("Error remove and hide items {}", ingredient)
+                        .add("Items must not be empty")
+                        .error()
+                        .post();
+                return;
+            }
+            JeiPlugin.hideItem(ingredient.getMatchingStacks());
+        }
+        List<ResourceLocation> recipesToRemove = new ArrayList<>();
+        for (IRecipe recipe : ForgeRegistries.RECIPES) {
+            if (recipe.getRegistryName() != null) {
+                for (IIngredient ingredient : ingredients) {
+                    if (ingredient.test(recipe.getRecipeOutput())) {
+                        recipesToRemove.add(recipe.getRegistryName());
```

Comment:
why have a `recipesToRemove` list instead of simply running `ReloadableRegistryManager.removeRegistryEntry(ForgeRegistries.RECIPES, recipe.getRegistryName());` here?